lib/sinon/util/fake_timers.js

```diff
@@ -2,17 +2,34 @@
 
 var llx = require("lolex");
 
-exports.useFakeTimers = function () {
-    var now;
-    var methods = Array.prototype.slice.call(arguments);
+/**
+ *
+ * @param config {number|Date|Object} the unix epoch value to install with (default 0) or
+ */
+exports.useFakeTimers = function (args) {
```

Comment:
I actually like `if ... else if ... else` because when you read it you know that once a certain condition has been met the execution breaks out of the block, not allowing for other blocks in the condition to evaluate and execute.

In the example you wrote, on first sight the second and third conditions might look also deceiving since: 
```javascript
> args = new Date()
2017-07-26T08:54:10.426Z
> args instanceof Date
true
> typeof args === "object"
true
```


The `Object.create()` doesn't fly and doesn't deep copy the array. sticking with the `for` loop for now.

On REPL do:
```javascript
let a= {a: ['a','b','c']}
undefined
> a
{ a: [ 'a', 'b', 'c' ] }
> let b = Object.create(null,aa)
undefined
> b
{}
```

I'm suggesting a shorter version of the `useFakeTimers` function, cleaner and shorter.
```javascript
exports.useFakeTimers = function (args) {
    var config = {};

    if (typeof args === "undefined" || args === null) {
        config.now = 0;
    } else if (typeof args === "number" || args instanceof Date) {
        config.now = args;
    } else if (args !== null && typeof args === "object") {
        var keys = Object.keys(args);
        for (var i = 0; i < keys.length; i++) {
            if (args.hasOwnProperty(keys[i])) {
                config[keys[i]] = args[keys[i]];
            }
        }
    } else {
        throw new TypeError("useFakeTimers expected epoch or config object." +
            " see https://github.com/sinonjs/sinon");
    }

    var clock = llx.install(config);
    clock.restore = clock.uninstall;
    return clock;
};
```

What do you think?